### PR TITLE
add cropZoomImgs option for fullscreen images

### PR DESCRIPTION
**Pull request by call-me-matt**
Adding an option to crop and zoom pictures so that no borders are shown if image ratio does not match the screen ratio.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ The following properties can be configured:
 | `autoDeleteImages`   | Defines if old images should be deleted, when they are no longer used in the slideshow (see 'imageCount'). Starred images will not be deleted.                       |
 | `showSender`         | When set to true, TeleFrame will show the name of the sender when the image is shown.                                                                                |
 | `showCaption`        | When set to true, TeleFrame will show the caption of the image when the image is shown.                                                                              |
-| `fullscreen`         | When set to true, TeleFrame will run in fullscreen mode.                                                                                                             |
```
Comment (call-me-matt) [inline README.md:100]: I overwrote the fullscreen entry, as it was in there twice.

**Comment by LukeSkywalker92** [comment]
Nice idea. Is it ready to merge?

**Comment by LukeSkywalker92** [comment]
but please create pull request to develop...see [CONTRIBUTING.md](https://github.com/LukeSkywalker92/TeleFrame/blob/master/.github/CONTRIBUTING.md)

**Comment by call-me-matt** [comment]
> Nice idea. Is it ready to merge?

I would feel better if someone could take a second look.

> but please create pull request to develop...see [CONTRIBUTING.md](https://github.com/LukeSkywalker92/TeleFrame/blob/master/.github/CONTRIBUTING.md)

Ah, I was not aware of that. Installed git-flow and tried to init, but it says there is no "next release" branch. Maybe you can help me to get everything in the needed format?

**Comment by call-me-matt** [comment]
to be continued here: https://github.com/LukeSkywalker92/TeleFrame/pull/130

**Pull request closed without merging**